ext/dom/tests/bug81433.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Bug #81433 (DOMElement::setIdAttribute(attr, true) called twice removes ID)
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+if (LIBXML_VERSION >= 20912) die('skip For libxml2 < 2.9.12 only');
```

Comment:
Is there any real reason to skip the test for libxml2 >= 2.9.12? This issue does not appear to be related to libxml2 at all, and for me the test fails with libxml2 2.9.12 without the patch.

Reply:
I removed this check in the following commit.